catboost/libs/options/catboost_options.cpp

```diff
@@ -259,7 +266,8 @@ namespace NCatboostOptions {
         if (lossFunction == ELossFunction::Quantile ||
             lossFunction == ELossFunction::MAE ||
             lossFunction == ELossFunction::LogLinQuantile ||
-            lossFunction == ELossFunction::MAPE)
+            lossFunction == ELossFunction::MAPE ||
+            lossFunction == ELossFunction::SMAPE)
```

Comment:
SMAPE defined with second order derivates, but here you ensure, that you never uses them
If loss has 2-order ders, then we can use Newton in leaves estimation

Reply:
Done!